AWS/EC2/AWS_-_Change_EC2_Instance_Size.yaml

```diff
@@ -0,0 +1,148 @@
+- defaultTab: nodes
+  description: |
+    Change EC2 Instance Size
+
+    This script does the following:
+    - Stops the selected EC2 instance.
+    - A script step ensures the instance is not running and performs error checking to verify that the new instance type is supported, then changes the type.
+    - Optionally starts the instance after successful change.
+
+    Requirements:
+    For detailed requirements see the corresponding solutions page in our docs at:  [https://docs.rundeck.com/learning/solutions](https://docs.rundeck.com/learning/solutions)
```

Comment:
when i click this link, it takes me to a 404

Reply:
strange... url works sometimes but not other times.  Adding the slash seems to help.